libs/application/templates/signature-collection/signature-list-creation/src/fields/ListsInOverview/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC } from 'react'
+import { ActionCard, Stack } from '@island.is/island-ui/core'
+import { FieldBaseProps } from '@island.is/application/types'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../lib/messages'
+import { CountryAreas } from '../../lib/constants'
+
+export const ListsInOverview: FC<FieldBaseProps> = ({ application }) => {
```

Comment:
for what purpose?

Reply:
Just questioning the use of a custom component as we are trying to avoid creating those if we can. So If this application deems this component necessary could we find a way make it reusable for other future applications?

Reply:
I dont think we need to worry about this one as it is really something very custom, is this case.